rs/bitcoin/ckbtc/minter/BUILD.bazel

```diff
@@ -194,3 +210,32 @@ rust_ic_test(
         "@crate_index//:serde_json",
     ],
 )
+
+rust_canbench(
+    name = "ckbtc_minter_canbench",
+    srcs = ["src/main.rs"],
+    compile_data = [":ckbtc_minter.did"],
+    data = ["test_resources/mainnet_events.mem.gz"],
+    env = {
+        "CANBENCH_STABLE_MEMORY_FILE": "$(rootpath :test_resources/mainnet_events.mem.gz)",
+        "CANBENCH_INIT_ARGS_HEX": "4449444c0b6b02fcb88b840301b0ced18403096e026c08f5d19a6b03e3a683c30404b6b8ba8405078ebdcfc608078fc0fec00a07b682e7c90b03f18c92d50b08cdb2eee10c076e686e056b0496bab03c068b98eaed020682b8f0da047f8390999e057f6d686e786e796c0bf5d19a6b03b6a9f6dc0271e3a683c30405b6b8ba840578f1f7fcf706688ebdcfc6087880b3d0ed090a8fc0fec00a07b682e7c90b03f18c92d50b08cdb2eee10c076b03a4d3bb457f8687acf9047fabdeaed9087f01000100176d61737465725f65636473615f7075626c69635f6b657903a086010000000000010affffffffffe00000010164000000000000000001640000000000000001010affffffffffe000020101010600000000",
```

Comment:
can you add a note about how this was generated? e.g. `didc --format hex '(...)'`

Reply:
Added comments